appveyor.yml

```diff
@@ -38,6 +38,7 @@ test_script:
   - ps:   |
             if ($env:PowerShellEdition -eq 'WindowsPowerShell') {
                 $modulePath = $env:PSModulePath.Split([System.IO.Path]::PathSeparator) | Where-Object { Test-Path $_} | Select-Object -First 1
+                if ( Test-Path "$modulePath\PSScriptAnalyzer" ) { Remove-Item -recurse -force "$modulePath\PSScriptAnalyzer" }
```

Comment:
Did you find that there was a `PSScriptAnalyzer` folder at some point? All 4 builds should run on fresh images (as if it was a new VM), therefore it would be surprising to me to find PSSA already being there.